manual/src/cmds/intf-c.etex

```diff
@@ -2500,9 +2500,13 @@ CAMLprim stub_gethostbyname(value vname)
 }
 \end{verbatim}
 
-During the time the domain lock is released, the thread-local variable
-"Caml_state" is set to "NULL". This can be used to determine if the
-thread currently holds its domain lock.
+The domain state variable "Caml_state" checks that the domain lock is
+held, either in debug mode or at key entry points of the C API.
```

Comment:
This reads wrong: a variable doesn't *do* anything. In any case, `Caml_state` only checks in debug mode, and you have another function to check in non-debug mode.